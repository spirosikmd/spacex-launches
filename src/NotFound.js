import React from 'react';
import Image from './Image';
import notFoundImage from './notFound.png';

const NotFound = () => {
  return <Image alt="Not found" src={notFoundImage} />;
};

export default React.memo(NotFound);
