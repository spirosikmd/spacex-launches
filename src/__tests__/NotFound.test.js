import { NotFound } from '../NotFound';

describe('NotFound', () => {
  it('renders', () => {
    expect(shallow(NotFound, {})).toMatchSnapshot();
  });
});
