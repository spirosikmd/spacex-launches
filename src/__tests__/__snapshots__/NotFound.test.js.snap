// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders 1`] = `
<WithStyles(Image)
  alt="Not found"
  src="notFound.png"
/>
`;
